subsys/fs/shell.c

```diff
@@ -17,12 +17,45 @@
 #include <inttypes.h>
 #include <limits.h>
 
+
+/* FAT */
+#ifdef CONFIG_FAT_FILESYSTEM_ELM
+#include <ff.h>
+#define FATFS_MNTP      "/RAM:"
+/* FatFs work area */
+FATFS fat_fs;
+/* mounting info */
+static struct fs_mount_t fatfs_mnt = {
+	.type = FS_FATFS,
+	.fs_data = &fat_fs,
+};
+#endif
+
+/* NFFS */
+#ifdef CONFIG_FILE_SYSTEM_NFFS
+#include <nffs/nffs.h>
+#define NFFS_MNTP       "/nffs"
+
+/* NFFS work area strcut */
+static struct nffs_flash_desc flash_desc;
+
+/* mounting info */
+static struct fs_mount_t nffs_mnt = {
+	.type = FS_NFFS,
+	.fs_data = &flash_desc,
+};
+#endif
+
 #define BUF_CNT 64
 
 #define MAX_PATH_LEN 128
 #define MAX_FILENAME_LEN 128
 #define MAX_INPUT_LEN 20
 
+
+#define SHELL_KERNEL    "fs"
```

Comment:
unused anywhere. "fs" is used explicite at the end of the file.

Reply:
Thanks, i missed using it...
Will change to use "SHELL_FS" in SHELL_REGISTER("SHELL_FS", fs_commands);

Reply:
@nvlsianpu : done